packages/vm/src/runTx.ts

```diff
@@ -300,6 +300,11 @@ async function _runTx(this: VM, opts: RunTxOpts): Promise<RunTxResult> {
   let fromAccount = await state.getAccount(caller)
   const { nonce, balance } = fromAccount
 
+  // EIP-3607: Reject transactions from senders with deployed code
```

Comment:
Also got some confirmation on this from the EIP:

> This EIP can be implemented as a soft fork because the new validity rules are a strict superset of the previous validity rules.